src/invidious/videos.cr

```diff
@@ -765,8 +765,8 @@ struct Video
     info["microformat"]?.try &.["playerMicroformatRenderer"]["isFamilySafe"]?.try &.as_bool || false
   end
 
-  def is_vr : Bool
-    info["streamingData"]?.try &.["adaptiveFormats"].as_a[0]?.try &.["projectionType"].as_s == "MESH" ? true : false || false
+  def is_vr : Bool?
+    return true if info.dig?("streamingData", "adaptiveFormats", 0, "projectionType").try &.as_s == "MESH"
```

Comment:
you should return directly the `==` comparison. if you do `return true if condition`, that will return nil in all cases where the condition is false. You can simply do that:
```
return (info.dig?(...).try &.as_s == "MESH")
```

`info.dig?(...).try &.as_s` will be either a `String` or `nil`, if the element was not found, and then `==` will alway return a `Boolean`.